sdk/messaging/azservicebus/internal/stress/tests/constant_detachment.go

```diff
@@ -30,19 +32,27 @@ func ConstantDetachment(remainingArgs []string) {
 	sender, err := client.NewSender(queueName, nil)
 	sc.PanicOnError("create a sender", err)
 
-	const maxMessages = 10000
+	const maxMessages = 20000
 
 	shared.MustGenerateMessages(sc, sender, maxMessages, 1024, stats)
 
+	time.Sleep(10 * time.Second)
```

Comment:
Can you document why we sleep here.  I don't see a problem with the sleep but understanding why it's here is carnal knowledge we need to document.

Reply:
Done.